polar-core/src/vm.rs

```diff
@@ -1593,13 +1668,10 @@ impl PolarVirtualMachine {
         self.append_goals(goals)
```

Comment:
GitHub won't let me comment on it, but we probably want to uncomment line 1654:
```rust
                // self.polar_log_mute = true;
```

Reply:
Sorry, I don't follow why or how that's related to these changes.

Reply:
It's something we did in one of the original unification PRs and forgot to undo before pushing to the `release-0.11.0-alpha` branch. It's a change in logging behavior and should be reverted prior to merging this to `main`.

Reply:
2d0879b4